defaults/main.yml

```diff
@@ -252,6 +252,13 @@ docker__storage_driver: 'overlay'
 docker__storage_options: {}
 
                                                                    # ]]]
+# .. envvar:: docker__arb_daemon_opts [[[
+#
+# Allows passing of arbitrary/unsupported configuration options to
+# 'daemon.json'.
+docker__arb_daemon_opts: {}
```

Comment:
I don't like using weird concatenations in variable names. How about `docker__custom_daemon_options`? I think that should be free to use.